src-plugins/medBinaryOperation/itkNotOperator.cpp

```diff
@@ -135,33 +99,30 @@ int itkNotOperator::update()
         }
         else
         {
-            qDebug() << description()
-                     <<", Error : pixel type not yet implemented ("
-                     << id
-                     << ")";
+            res = medAbstractProcess::PIXEL_TYPE;
         }
     }
     return res;
 }
 
+
 template <class ImageType> int itkNotOperator::run()
 {
-    if ( !d->input->data() )
+    if ( !m_inputA->data() )
     {
         return DTK_FAILURE;
     }
-    typename ImageType::Pointer image = dynamic_cast<ImageType  *> ( ( itk::Object* ) ( d->input->data() )) ;
+    typename ImageType::Pointer image = dynamic_cast<ImageType  *> ( ( itk::Object* ) ( m_inputA->data() )) ;
 
     if (!image)
```

Comment:
Yes, but why do this cast could bug? If it would never happen, that's not needed, no?